app/hooks/use-socket.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from "react";
 import { socket } from "@/socket";
 
-export default function useSocket() {
+export const useSocket = () => {
```

Comment:
now why'd you go and do something like that? I know you don't like default exports (I've come to agree with you). you can just change this to this (but don't forget the trailing semicolon):
```suggestion
export function useSocket() {
```